scripts/emulate_app.sh

```diff
@@ -249,46 +243,53 @@ else
 
     echo -e "\nWaiting for AVD to boot..."
     wait_for $timeout_seconds check_boot
-    echo -e "\nWaiting for AVD to be online..."
     wait_for $timeout_seconds check_device_online
     echo $(adb -s emulator-5554 emu avd name | head -1)
     echo "Device online" && sleep 1
 
-    echo -e "\nInstalling APKs..."
-    i=0
-    step_complete=false
-    until [[ $step_complete == true ]]; do
-        if adb -s emulator-5554 install -r -t -d --abi "${abi}" \
-                "app/build/outputs/apk/debug/app-${abi}-debug.apk" &> "${test_report_dir}/apk_installation.txt"; then
-            step_complete=true
-            echo "Successfully installed APKs"
-        fi              
-        if [[ $i -ge 100 ]]; then
-            echo "Error: Failed to install APKs" >&2
-            echo "For more information see 'android/${test_report_dir}/apk_installation.txt'" >&2
-            exit 1
-        fi
-        i=$((i+1))
-        sleep 1
-    done
+    if "${boot_app}"; then
```

Comment:
can't remember exactly the issue or have the source right now but i have read about issues with boolean in `if` statements with bash, can you add the 'equals true` part like here:
https://github.com/zingolabs/zingo-mobile/pull/330/files/53aef2b69b6c1e1aae75a54e3725edb11a997cd2..5eb69bf6e343c99e65869837ff3b8d6ba816505b#diff-c978b23d882c1e0ea4a509c0d51c9294acee8ba79ec8a92a0650696ee17b18e7R166